index.js

```diff
@@ -3,6 +3,38 @@
 const fp = require('fastify-plugin')
 var pg = require('pg')
 
+function transactionHelper (query, values, cb = null) {
+  return new Promise((resolve, reject) => {
```

Comment:
Could you please have another look?

Sorry, I lack experience with promises and callbacks. Another issue is that I am having trouble building the project since I am on windows so just using here to run the tests 😄 

Apologies for spamming